### ISSUE TITLE
Refactor `def train_val_test_split(df, cols):` to be more configurable

### ISSUE THREAD

**Issue by tallamjr** [tallamjr/astronet#43]
At the moment, the train, validation and test set are determined by hard-coded amounts inside `+70 astronet.t2.utils`:
```python
...
 70 def train_val_test_split(df, cols):
 71
 72     features = df[cols]
 73     # column_indices = {name: i for i, name in enumerate(features.columns)}
 74
 75     n = len(df)
 76     df_train = df[0 : int(n * 0.8)].copy()
 77     df_val = df[int(n * 0.8) : int(n * 0.95)].copy()
 78     df_test = df[int(n * 0.95) :].copy()
 79
 80     num_features = features.shape[1]
 81
 82     return df_train, df_val, df_test, num_features
```
It would be desirable able to have this determined by parameters in the function call perhaps.

This will link to #44 here where one would like to compare against benchmark results on the data. To have a fair comparison, the same dataset, and data split should be use.

Another major motivation of this is to allow for optimisation via hyperparameter tuning to be done using a hold-out validation set and `keras.callbacks`; then when it comes to training, to train on the full "_training_" set, i.e **train** + **validation** (still leaving test set out completely until evaluation afterwards with `astronet.t2.evaluate.py`)

Linked issues:
- #39 
- #44

Refs:
- https://github.com/keras-team/keras/issues/1753
